app/server/utils/metrics/participant_stats.py

```diff
@@ -57,16 +62,21 @@ def __init__(self, group_strategy, timeseries_unit = 'day'):
             aggregated_query_actions=[FORMAT_AGGREGATE_METRICS],
             total_query_actions=[GET_FIRST],
         ))
-    
-        active_users_timeseries_query = db.session.query(func.count(func.distinct(CreditTransfer.sender_user_id)).label('volume'),
-                func.date_trunc(self.timeseries_unit, CreditTransfer.created).label('date'), group_strategy.group_by_column).group_by(func.date_trunc(self.timeseries_unit, CreditTransfer.created))
-        aggregated_active_users_query = db.session.query(func.count(func.distinct(CreditTransfer.sender_user_id)).label('volume'), group_strategy.group_by_column)
+
+        if group_strategy:
+            active_users_timeseries_query = group_strategy.build_query_group_by_with_join(db.session.query(func.count(func.distinct(CreditTransfer.sender_user_id)).label('volume'),
+                    func.date_trunc(self.timeseries_unit, CreditTransfer.created).label('date'), group_strategy.group_by_column).group_by(func.date_trunc(self.timeseries_unit, CreditTransfer.created)), CreditTransfer)
```

Comment:
could consider turning `db.session.query(func.count(func.distinct(CreditTransfer.sender_user_id)).label('volume'), func.date_trunc(self.timeseries_unit, CreditTransfer.created).label('date')`  etc into vars to DRY up a little more, but NBD either way